server/views/pages/conditions/select-conditions/index.njk

```diff
@@ -22,14 +22,14 @@
 
           {% macro conditionNameInput(conditionType, value) %}
             {{ govukInput({
-              id: conditionType + "_conditionName",
-              name: "conditionName[" + conditionType + "]",
+              id: conditionType + "_conditionNames",
+              name: "conditionNames[" + conditionType + "]",
```

Comment:
Plural is correct 👍 
(ie. its a map of condition names, keyed by the condition type)